megamek/src/megamek/client/ui/swing/boardview/BoardView1.java

```diff
@@ -2451,8 +2464,12 @@ public BufferedImage getEntireBoardImage(boolean ignoreUnits) {
                 }
             }
         }
-
         boardGraph.dispose();
+
+        // Restore the zoom setting
+        zoomIndex = oldZoom;
+    	zoom();
```

Comment:
Whitespace is off starting here it looks.

Reply:
My code formatter was improperly configured and it was inserting some tabs.  I saw this during the commit and I should've fixed it then.  I've fixed it now (there were a few other places too besides this one).